### ISSUE TITLE
assimp351 builds failed?

### ISSUE THREAD

**Issue by traversaro** [conda-forge/hpp-fcl-feedstock#96]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

It seems that many assimp351 builds failed with error (see https://github.com/conda-forge/hpp-fcl-feedstock/runs/17188625744):
~~~
2023-09-27T16:55:35.3503917Z [  4%] Generating Doxygen C++ documentation
2023-09-27T16:55:35.3510946Z cd $SRC_DIR/build/python && $PREFIX/bin/python $SRC_DIR/doc/python/doxygen_xml_parser.py $SRC_DIR/build/doc/doxygen-xml/index.xml $SRC_DIR/build/python/doxygen_autodoc > $SRC_DIR/build/python/doxygen_autodoc.log
2023-09-27T16:55:35.4408989Z Traceback (most recent call last):
2023-09-27T16:55:35.4411383Z   File "/home/conda/feedstock_root/build_artifacts/hpp-fcl_1695833542832/work/doc/python/doxygen_xml_parser.py", line 893, in <module>
2023-09-27T16:55:35.4415543Z     index.parseCompound()
2023-09-27T16:55:35.4417736Z   File "/home/conda/feedstock_root/build_artifacts/hpp-fcl_1695833542832/work/doc/python/doxygen_xml_parser.py", line 697, in parseCompound
2023-09-27T16:55:35.4424209Z     obj = ClassCompound(compound, self)
2023-09-27T16:55:35.4426039Z   File "/home/conda/feedstock_root/build_artifacts/hpp-fcl_1695833542832/work/doc/python/doxygen_xml_parser.py", line 417, in __init__
2023-09-27T16:55:35.4431964Z     self._templateParams(self.definition.find("templateparamlist"))
2023-09-27T16:55:35.4436051Z   File "/home/conda/feedstock_root/build_artifacts/hpp-fcl_1695833542832/work/doc/python/doxygen_xml_parser.py", line 456, in _templateParams
2023-09-27T16:55:35.4439642Z     [_templateParamToDict(param) for param in tpl.iterchildren(tag="param")]
2023-09-27T16:55:35.4440681Z   File "/home/conda/feedstock_root/build_artifacts/hpp-fcl_1695833542832/work/doc/python/doxygen_xml_parser.py", line 456, in <listcomp>
2023-09-27T16:55:35.4445654Z     [_templateParamToDict(param) for param in tpl.iterchildren(tag="param")]
2023-09-27T16:55:35.4449515Z   File "/home/conda/feedstock_root/build_artifacts/hpp-fcl_1695833542832/work/doc/python/doxygen_xml_parser.py", line 102, in _templateParamToDict
2023-09-27T16:55:35.4453050Z     typetext += c.text
2023-09-27T16:55:35.4489555Z TypeError: unsupported operand type(s) for +=: 'NoneType' and 'str'
~~~

And this is blocking the assimp351 migration in https://conda-forge.org/status/#assimp531 .

### Installed packages

```shell
.
```

### Environment info

```shell
.
```

**Comment by jcarpent** [comment]
I will do a new release of hpp-fcl then.
Thanks @traversaro for the report.

**Comment by jcarpent** [comment]
Solved via #97
